website/pages/index.js

```diff
@@ -61,6 +62,15 @@ function Home() {
   const context = useContext(DocusaurusContext);
   const {siteConfig = {}} = context;
 
+  const feedbackUrl = `${siteConfig.baseUrl}feedback/`;
+  const gettingStartedUrl = `${siteConfig.baseUrl}docs/installation`;
+
+  useEffect(() => {
+    // Prefetch feedback pages & getting started pages
+    __docusaurus.prefetch(feedbackUrl) &&
```

Comment:
I get why we write it like this, but I think we should put them on separate lines. The difference is not very significant.